content/webapp/pages/exhibition-guide.tsx

```diff
@@ -320,13 +320,7 @@ const Stops: FC<StopsProps> = ({ stops, type }) => {
               <VideoEmbed embedUrl={bsl.embedUrl} />
             )}
           </Stop>
-        ) : (
-          <Stop key={index} id={dasherizeShorten(title)}>
-            <span className={font('intb', 5)}>
-              {number}. {title}
-            </span>
-          </Stop>
-        );
+        ) : null;
```

Comment:
Might be worth a comment here noting that it's an explicit decision to omit stops that don't have relevant content.